<launch>

  <node pkg="minitask4" type="patrol.py" name="move_base_seq" output="screen">
        <rosparam param="p_seq">[4.3,-2.2,0,5,2.1,0,1.3,2.5,0,0.2,0.8,0,1.2,-0.9,0,2.4,-0.1,0]</rosparam>   
        <rosparam param="yea_seq">[0.7, -1, -0.9,-0.5, 0.3,-0.4]</rosparam>

  </node>

</launch>
